sympy/solvers/tests/test_ode.py

```diff
@@ -2552,3 +2552,14 @@ def test_issue_7093():
     sol = Eq(f(x), C1 - 2*x*sqrt(x**3)/5)
     eq = Derivative(f(x), x)**2 - x**3
     assert dsolve(eq) == sol and checkodesol(eq, sol) == (True, 0)
+
+
+def test_dsolve_linsystem_symbol():
+    x, y = symbols('x, y', function=True)
+    t = Symbol('t')
```

Comment:
near top of file has `f, g, h` functions: use those instead.  `x`  is also defined as independent variable.  See e.g., `test_old_ode_tests()`.  (Looks like lots of other tests don't do this so probably not a big deal).